source/tools.rst

```diff
@@ -101,6 +101,12 @@ Once python and pip are set up, you can install Sphinx by using the following co
 
   $ pip install Sphinx
 
+Run the following command to setup Sphinx's environmental variables:
+
+.. code-block:: console
+
+  $ sphinx-quickstart
```

Comment:
This command is only used to start a new documentation project. I don't think this part of the instructions are for starting a new project, just getting an existing one up and running.

Reply:
removed.